queries.sql

```diff
@@ -25,4 +25,32 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 
 //animals with a weight between 10.4kg and 17.3kg
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+//How many animals are there?
+SELECT COUNT(name) FROM animals;
+
+//How many animals have never tried to escape?
+SELECT COUNT(escape_attempts) FROM animals
+WHERE escape_attempts = 0;
+
+//What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+
+
+//Who escapes the most, neutered or not neutered animals?
+SELECT MAX(escape_attempts) as max_escapes FROM animals WHERE neutered = 'true';
+SELECT MAX(escape_attempts) as max_escapes FROM animals WHERE neutered = 'false';
```

Comment:
- Kindly group your result to indicate who escape most between neutered and not neutered animals.